src/scripts.js

```diff
@@ -91,32 +93,11 @@ function viewFavorites() {
   } else {
     favButton.innerHTML = 'Refresh Favorites'
     cardArea.innerHTML = '';
-    user.favoriteRecipes.forEach(recipe => {
```

Comment:
Boom - this is great! way to consolidate the function - this is going to be great for DomUpdates!